sequence_similarity.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+import pandas as pd
+from datasets import load_dataset
+
+dataset = load_dataset("roszcz/internship-midi-data-science", split="train")
+
+record = dataset[0]
+df = pd.DataFrame(record["notes"])
+print(df.head())
+
+
+def cos_sim_score(sequence: pd.DataFrame, window: pd.DataFrame) -> float:
+    """
+    Calculating cosine similarity between sequence and window
+    Args:
+        sequence (pd.DataFrame): input sequence
+        window (pd.DataFrame): subset of rolling window
+    Returns:
+        float: cosine similarity score
+    """
```

Comment:
```suggestion
def cos_sim_score(sequence_a: pd.DataFrame, sequence_b: pd.DataFrame) -> float:
    """
    Calculating cosine similarity between two sequences
    Args:
        sequence_a (pd.DataFrame): first sequence
        sequence_b (pd.DataFrame): second sequence
    Returns:
        float: cosine similarity score
    """
```

This is great, but name "window" doesn't really make sense from the point of view of this function - it measures distance between any two sequences and it's only in your specific use case that the second sequence is a "window" moving over the signal.